less/plugins/adapt-contrib-slider/slider.less

```diff
@@ -1,77 +1,99 @@
-.slider-component {
+.slider {
+  // --------------------------------------------------
+  // Numbers
+  // --------------------------------------------------
+  &__number-container {
+    min-height: 2.5rem;
+  }
 
-    .slider-scaler-wrapper {
+  &__number,
+  &__number-selection,
+  &__number-model-answer {
+    height: 2.5rem;
+    width: 2.5rem;
+    line-height: 2.5;
+    text-align: center;
+    border-radius: 50%;
+  }
 
-        .slider-scaler {
-          background-color: @item-body-text-color;
-        }
-    }
+  // &__number {
+  //   background-color: @item-color;
+  //   color: @item-color-inverted;
+  // }
 
-    .slider-model-answer,
-    .slider-scale-marker {
-        color: @item-text-color;
-        background-color: @item-color;
-    }
+  &__widget:not(.is-disabled) .slider__number {
+    cursor: pointer;
 
-    .slider-line {
-        background-color: @item-body-text-color;
-    }
-}
+    // .no-touch &:hover {
+    //   background-color: @item-color-hover;
+    //   color: @item-color-inverted-hover;
+    //   .transition(background-color @duration ease-in, color @duration ease-in;);
+    // }
+  }
 
-.slider-widget {
-    .slider-holder {
-        .rangeslider__handle {
-          background-color: @item-color;
-          &:hover {
-            .no-touch & {
-              background-color: @item-color-hover;
-            }
-          }
-        }
-        .rangeslider__fill {
-          background-color: @item-color;
-        }
-        .rangeslider {
-          background-color: @background-color;
-        }
+  &__number-selection {
+    background-color: @item-color;
+    color: @item-color-inverted;
+    // background-color: @item-color-selected;
```

Comment:
Should these be deleted?

Reply:
I recommend leaving comment in as code allows different style of button.